docs/source/settings.rst

```diff
@@ -359,6 +367,11 @@ speed up server boot times. Although, if you defer application loading
 to each worker process, you can reload your application code easily by
 restarting workers.
 
+.. note::
```

Comment:
The note explicitly mentions the gevent and eventlet workers. Or did you mean remove the note directive and make it a normal paragraph?